recipe/meta.yaml

```diff
@@ -1,7 +1,7 @@
 {% set name = "qdarkstyle" %}
 {% set package_name = "QDarkStyle" %}
-{% set version = "3.0.2" %}
-{% set hash = "55d149cf5f40ee297397f1818e091118cefb855a4a9c5c38566c47acd2d8c7ae" %}
+{% set version = "3.2.3" %}
+{% set hash = "0c0b7f74a6e92121008992b369bab60468157db1c02cd30d64a5e9a3b402f1ae" %}
 
 package:
   name: {{ name|lower }}
```

Comment:
the `fn:` thing can be removed. It's a legacy feature which isn't used any more: https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#filename
(it doesn't really affect anything, just for tidyness really)

Reply:
RE your other comment, I do think it's safe to leave noarch for this package, but if that ever changes I don't mind removing it. As far as the `fn`, I know it's not necessary, but since it was already there, I left it just in case a user expects that filename syntax for whatever reason. If it does more harm than good being left in the recipe, then I'll gladly open another PR to remove it. Please let me know!

Reply:
> do think it's safe

ok, sounds good, but was just wondering what the technical reasons were